apps/server/src/shared/domain/entity/external-tools/external-tool/config/lti11-tool-config.ts

```diff
@@ -10,7 +10,7 @@ export class Lti11ToolConfig extends ExternalToolConfig {
 	key: string;
 
 	@Property()
-	secret: string;
+	secret?: string;
```

Comment:
Is there a usecase or need for an Lti11ToolConfig to exist that doesnt have a secret? since it leads to an error anyway, it seems to me we should find a way to prevent them from being created without secret in the firstplace?